operations/patch_module.go

```diff
@@ -34,16 +33,11 @@ func PatchSetModule() cli.Command {
 			committedOnly := c.Bool(committedFlagName)
 			args := c.Args()
 
-			ctx, cancel := context.WithCancel(context.Background())
-			defer cancel()
-
 			conf, err := NewClientSettings(confPath)
 			if err != nil {
 				return errors.Wrap(err, "problem loading configuration")
 			}
 
-			_ = conf.GetRestCommunicator(ctx)
```

Comment:
I think you want to do this so you end up getting the warning about needing to upgrade the agent.

Reply:
I re-added `GetRestCommunicator()` and included `Close()`.